guidance_pipeline/main.go

```diff
@@ -65,7 +130,7 @@ func makeMapOfRegs(header string, records [][]string) map[string][]Guidance {
 
 func writeRegsToFile(header string, regs map[string][]Guidance) error {
 	for key, reg := range regs {
-		filename := formatFilename(key)
+		filename := formatFilename(*outputDirectory, key)
```

Comment:
Give more context in the error states by adding things like this filename. Things like `fmt.Errorf("%s %s", filename, err)` work or a special Error type if it makes sense.